src/Clue.js

```diff
@@ -0,0 +1,26 @@
+class Clue {
+  constructor(clueObj, dailyDoubleBoolean) {
+    this.question = clueObj.question
+    this.answer = clueObj.answer
+    this.dailyDoubleBoolean = dailyDoubleBoolean;
+    this.pValue = this.updatePointValue(clueObj.pointValue);
```

Comment:
I'm a bit confused about this line. The point value is a method but also in that method we are multiplying the point value by 2?